doc/how-to-run-in-windows.rst

```diff
@@ -0,0 +1,352 @@
+Haskell, GHC and Cabal in Windows
+=================================
+
+The Windows toolchain and environment
+-------------------------------------
+
+`MinGW <https://www.mingw-w64.org/>`_ is a set of software packages for the
+Windows platform that enable development and compilation targeting the native
+Windows API. MinGW does not offer a complete developer environment so it usually
+is coupled with `MSYS2 <https://www.msys2.org/>`_ which provides many UNIX-like
+tools such as ``bash``, ``make`` or the package manager ``pacman``.
+
+MSYS2 comes with different environments. You can learn more about the different
+environments in the `MSYS2 documentation
+<https://www.msys2.org/docs/environments/>`_. We will be using ``UCRT64`` in the

```

Comment:
We should be using `MINGW64` in the examples, since this is the default environment for ghcup and stack.

Reply:
I thought you explicitly recommended UCRT64 the other day? Though I can’t find the post, so I may be misremembering.

Reply:
In the new version I mention explicitly that GHCup uses MINGW64 by default. To my understanding, MSYS2 already explicitly stated that the default should be UCRT64, that's why I use it and why I advertised it here, but indeed, the default is MINGW64. We will have to update the documentation once GHCup decides to move to UCRT64.